src/components/oif/vancouver/screen.vue

```diff
@@ -0,0 +1,275 @@
+                    <div class="col-9">
+                        <div class="text-uppercase">{{ schedule.state.track.name }}</div>
+                    </div>
+                    <div class="col-3 text-right">                    
+                        <div class="time">{{ schedule.getDate(schedule.state.now).format('h:mmA') }}</div>
+                    </div>
+                </div>            
+            </div>                    
+            <div class="container-fluid px-7" v-bind:class="schedule.state.events.curr ? 'pt-3' : 'pt-6'">
+                <div class="row">
+                    <div class="col-9">
+                        <div class="text-uppercase" v-bind:style="roomStyle">{{ formatRoomName(schedule.room.name) }}</div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+
+        <event :schedule="schedule" :event="schedule.state.events.curr" v-if="schedule.state.events.curr"></event>
+        
+        <event :schedule="schedule" :event="schedule.state.events.next" :next=true 
+            v-if="schedule.state.events.next && schedule.isToday(schedule.state.events.next.start_date)" 
+            v-bind:class="{ '': schedule.state.events.curr }"></event>
+
+        <div class="container-fluid" v-else-if="!schedule.state.events.curr">
+            <div class="row p-7 no-presentations">
+                <div class="col-12 text-left">
+                    All presentations are finished for today
+                </div>
+            </div>
+        </div>        
+
+    </div>
+</template>
+
+<script>    
+
+    import 'assets/css/ocp/2019/global/theme.scss'
+
+    import Event from './event.vue'
+    import moment from 'moment'
+
+    import { mapGetters } from 'vuex'    
+
+
+    export default {
+        computed: {
+            ...mapGetters({
+                schedule: 'schedule'
+            }),
+            roomStyle: function() {
+                return {
+                    'font-size': this.schedule.room.name.length >= 10 ? '3.5rem' : '4.5rem',
+                    'line-height': this.schedule.room.name.length >= 10 ? '3.5rem' : '4.5rem',                    
+                }
+            },
+            trackStyle: function() {
+                var color = '#ffffff';
+                if (this.schedule.state.track &&
+                    this.schedule.state.track.color) {
+                    color = this.schedule.state.track.color;
+                }
+                return { 'color': color, 'font-size': this.schedule.room.name.length >= 15 ? '2.5rem' : '2rem' }
+            },
+            summitScheduleUrl: function() {
+                return 'https://2022ocpglobal.fnvirtual.app/a/schedule'
+            }
+        },
+        methods: {
+            formatRoomName(name) {
+                if (name.match(/^\d/)) {
+                    return name.replace(/\s/g, '');
+                }
+                return name
+            },
+            scheduleIsOver() {
```

Comment:
this should be a computed property

  computed: {
        ...mapGetters({
            schedule: 'schedule'
        }),
        roomStyle: function() {
            return {
                'font-size': this.schedule.room.name.length >= 10 ? '4.5rem' : '10.5rem',
                'line-height': '4rem',
            }
        },
        trackStyle: function() {
            var color = '#FFFFFF';
            console.log(this.schedule.state.track)
            if (this.schedule.state.track &&
                this.schedule.state.track.color) {
                color = this.schedule.state.track.color;
            }
            return { 'backgroundColor': color }
        },
        summitScheduleUrl: function() {
            return 'https://2022ocpglobal.fnvirtual.app/a/schedule'
        },
        **scheduleIsOver: function(){
            return !this.schedule.state.events.curr;
        }**
    },